inst/scripts/make-metadata.R

```diff
@@ -58,7 +61,8 @@ main.data <- data.frame(
     "TXT",
     "TSV",
     "RDA",
-    "RDA"
+    "RDA",
+    "RDS"
```

Comment:
Don't remember what they do for RDS... let's store the data as RDA. Do you already have the `ref_mouse_atlas.rda` file?

Reply:
I think I just have it in rds form

Reply:
Update: I now have it in RDA form: https://github.com/rnabioco/scRNA-seq-Cell-Ref-Matrix/blob/master/atlas/musMusculus/MouseAtlas.rda

Reply:
See 93daea8